src/smt/smt_context.h

```diff
@@ -227,6 +227,17 @@ namespace smt {
 
         // -----------------------------------
         //
+        // Theory case split
+        //
+        // -----------------------------------
+        typedef int_hashtable<int_hash, default_eq<int> > int_set;
```

Comment:
we mainly use uint_set for sets over dense domains, such as literals. Hash-tables may have less memory overhead, but tend to not perform as well. So hash-tables make more sense if we have to maintain a large number of them simultaneously.